libvast/vast/system/index.hpp

```diff
@@ -36,6 +36,11 @@
 
 namespace vast::system {
 
+enum class keep_original_partition : bool {
```

Comment:
Please add an explanatory comment detailing what this type does. It's public API, after all.